LEAF_Nexus/sources/Group.php

```diff
@@ -402,6 +412,9 @@ public function getGroupLeader($groupID)
 
     public function listGroupPositions($groupID)
     {
+        if(!is_numeric($groupID)){
+            return array();
```

Comment:
Since `Exceptions` are thrown elsewhere when the input is invalid, instead of returning an empty array, it should throw `new Exception('invalid group');`